modules/ROOT/pages/create-application.adoc

```diff
@@ -58,14 +58,11 @@ Congratulations! You have successfully created your first process and your first
 [NOTE]
 ====
 
-Verify that the application has been created in Bonita Portal.
+Verify that the application is available.
 
-. Click on the Portal icon image:images/getting-started-tutorial/create-application/portal-icon.png[Portal icon] in the Bonita Studio coolbar.
-. In the upper right corner of the window, click on the *User* drop down menu
-. Select *Administrator*
-. Click on the *Applications* tab
+. Click on the Applications icon image:images/getting-started-tutorial/create-application/applications-icon.png[Applications icon] in the Bonita Studio coolbar.
+. You will see the application _Claims_ that you have just deployed listed in Bonita Application Directory. If you click on the URL, _My Dashboard_ page will be displayed in your brower.
```

Comment:
brower ? or browser?

Reply:
![Bowser](https://www.google.com/url?sa=i&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FBowser_(character)&psig=AOvVaw1niYtFiHSaCpZs_Ke7BKRf&ust=1631708294759000&source=images&cd=vfe&ved=0CAkQjRxqFwoTCOjbqbm5_vICFQAAAAAdAAAAABAD)

Reply:
I modified it to browser